addons/web/static/src/views/view_dialogs/select_create_dialog.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="web.SelectCreateDialog" owl="1">
-        <Dialog title="props.title">
+        <Dialog title="props.title" contentClass="'o_select_create_dialog'">
```

Comment:
hmm, I think that the default should be the other way around: no padding by default, and the content can add margin if necessary. this allows for maximal flexibility/reuse.